android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -20,6 +20,7 @@ import mozilla.components.browser.engine.gecko.media.GeckoMediaDelegate
 import mozilla.components.browser.engine.gecko.mediasession.GeckoMediaSessionDelegate
 import mozilla.components.browser.engine.gecko.permission.GeckoPermissionRequest
 import mozilla.components.browser.engine.gecko.prompt.GeckoPromptDelegate
+import mozilla.components.browser.engine.gecko.shopping.GeckoProductRecommendation
```

Comment:
Should the var name include `Shopping` like in `GeckoShoppingProductAnalysisResult`, or can `GeckoShoppingProductAnalysisResult` be `GeckoProductAnalysis`

Reply:
Sure, let's unify them.

I added `Shopping` because I wanted to make it clear what kind of product we are talking about, or rather which feature is this a part of. The word "Fakespot" seemed too Gecko-specific, so I thought "Shopping" would be nice. The package _is_ already called `shopping`, so maybe that's enough. I'll remove the word `shopping` from the `ProductAnalysis`